### ISSUE TITLE
InterproScan annotations not loaded correctly

### ISSUE THREAD

**Issue by bradfordcondon** [statonlab/hardwoods_site#438]
https://www.hardwoodgenomics.org/bio_data/207500?tripal_pane=group_interpro

![screen shot 2018-11-06 at 3 44 36 pm](https://user-images.githubusercontent.com/7063154/48092549-f7202800-e1da-11e8-869b-1f5a5f310937.png)

Note the IPR Term "None" values.  Note the accession pane has NO interproscan annotations.

The IPR loader does in fact associate the terms in feature_cvterm.  

conclusion: we dont have interproscan cvterms loaded live and/or they werent accessible to the loader.

**Comment by almasaeed2010** [comment]
It seems the code is expecting this to happen since it is display the word `None` rather than failing. So are we sure this is a bug with the module?

**Comment by bradfordcondon** [comment]
no, i think its not a bug with the module, but rather a problem with us not having the IPR cvterms loaded when the data imported.

**Comment by mestato** [comment]
This is related to the db source. Some DB results are returned by IPS but they aren't actually part of the IPS database (no term id, no description). IE, all results from sources, Coils, TMHMM, Panther, will never have an ipr term or description. if you search those on the interpro site, it says "unintegrated" and links you to the source db. Results from sources such as pfam will have ips terms and descriptions.

Would be nice to figure out all sources without ipr terms and create links to the tools just so users could get some additional information, for example see this, could have links to coils and tmhmm

<img width="855" alt="screen shot 2019-03-06 at 2 33 09 pm" src="https://user-images.githubusercontent.com/508122/53908214-d34fe000-401c-11e9-9e1d-f9dc1f5cceaa.png">
